cms_index.php

```diff
@@ -0,0 +1,109 @@
+<?php
+require_once 'php/dbConnection.php';
+require_once 'php/functions.php';
+
+$db = getdbConnection();
```

Comment:
getdbConnection should be camel case getDbConnection

Reply:
Done